libclc/CMakeLists.txt

```diff
@@ -72,7 +72,7 @@ else()
   # Note that we check this later (for both build types) but we can provide a
   # more useful error message when built in-tree. We assume that LLVM tools are
   # always available so don't warn here.
-  if( NOT clang IN_LIST LLVM_ENABLE_PROJECTS )
+  if( NOT LLVM_RUNTIMES_BUILD AND NOT clang IN_LIST LLVM_ENABLE_PROJECTS )
```

Comment:
I'd quite like to keep this check/error if possible. The error you get if you forget `-LLVM_ENABLE_PROJECTS=clang` is horrible. Do you think it would it acceptable to pass through `LLVM_ENABLE_PROJECTS` to the runtimes build, @jhuber6?

Reply:
Thanks. Yes I suppose in the absence of a higher-level warning like you describe, we could probably use `USE_TOOLCHAIN` to accomplish the same thing in a runtimes build.

With any luck this is all temporary, and a runtimes build will allow us to just take the cmake c compiler.